daprdocs/content/en/developing-applications/building-blocks/service-invocation/howto-invoke-services-grpc.md

```diff
@@ -0,0 +1,238 @@
+---
+type: docs
+title: "How-To: Invoke services using gRPC"
```

Comment:
Maybe this is better called "How-To: Invoke existing services using gRPC"

Reply:
There's no difference between existing and non-existing. This is how to use Dapr and gRPC.